packages/api/app/launchpadAuth.js

```diff
@@ -49,9 +49,9 @@ async function ensureLaunchpadAPIAuthorized(req, res, next) {
     }
   } catch (error) {
     if (error instanceof RecordDoesNotExist) {
-      const passphrase = await secretsManager().getSecretValue({
-        SecretId: process.env.launchpad_passphrase_secret_name
-      }).promise();
+      const passphrase = await getSecretString(
```

Comment:
So previously this code here wasn't pulling the `SecretString` out of the response, which I assume is part of the fix here?